### ISSUE TITLE
Make it work with python 3.6

### ISSUE THREAD

**Issue by tristanlatr** [tristanlatr/astuce#3]

**Comment by pawamoy** [comment]
Do you need this for pydoctor?

**Comment by tristanlatr** [comment]
Yes that would be good, but maybe we can increase the required python version of `pydoctor` to 3.7 also. It depends how much annoyance it will be to handle the AST differences.

**Comment by tristanlatr** [comment]
One thing we can do to easy this is to transform all Str, Bytes, Num and other deprecated nodes by their Constant counterparts. I’m not sure is there are other notable changes from 3.6.
